src/views/components/landing-top.html

```diff
@@ -14,7 +14,7 @@ <h2>free, fast, and reliable</h2>
 					{{#if headingMessage === 0}}
 					<div slide-in-out>serving ~ {{_.formatHits(networkStats.hits.total)}} billion requests / month</div>
 					{{elseif headingMessage === 1}}
-					<div slide-in-out>serving ~ {{_.formatMB(networkStats.megabytes.total)}} TB / month</div>
+					<div slide-in-out>serving ~ {{_.formatMB(networkStats.megabytes.total)}} TiB / month</div>
```

Comment:
is this meant to be terabytes when it's `megabytes.total`?

Reply:
Yes, I renamed the function to `formatMiB2TiB` to make it more clear what it does.